rollrus_test.go

```diff
@@ -165,6 +165,20 @@ func TestTriggerLevels(t *testing.T) {
 	}
 }
 
+func TestWithMinLevel(t *testing.T) {
```

Comment:
I'd like to see another test proving that it doesn't just include all of the levels, always, which would pass this test.

Reply:
Yes, I agree. I just wanted to validate the idea before writing too many tests. :)

I updated my PR with more tests. Let me know if you had a particular kind of test in mind, apart from those, and I'll try to implement them.